src/components/Alert.tsx

```diff
@@ -23,11 +24,11 @@ const getTitle = (props: AlertProps) => {
 	if (props.prefix !== undefined) {
 		return props.prefix;
 	}
-	return props.danger ? 'Darn!' :
-		props.warning ? 'Warning!' :
-		props.success ? 'Great!' :
-		props.info ? 'Hey!' :
-		'';
+	return props.danger
+		? 'Darn!'
+		: props.warning
+			? 'Warning!'
+			: props.success ? 'Great!' : props.info ? 'Hey!' : '';
```

Comment:
I hate how prettier formats multi-ternary expressions.
They have an issue to track this... but still no progress https://github.com/prettier/prettier/issues/737